### PR TITLE
Doc/update readme

### LINKED ISSUE #16: Update README
Looks like this README could be updated, happy to do it myself!

### PR DESCRIPTION
**Pull request by tekktrik**
Resolves #16 by updating README

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 repos:
 -   repo: https://github.com/fsfe/reuse-tool
-    rev: latest
+    rev: v0.14.0
```
Comment (makermelissa) [inline .pre-commit-config.yaml:7]: What's the reasoning behind changing this to v0.14.0? I may have missed some conversation about this.

File: install.sh [hunk 2]
```diff
@@ -20,5 +20,5 @@ fi
 sudo apt-get update
 sudo apt-get install libudev-dev libusb-1.0
 sudo apt-get install -y gettext
-pip install -r requirements.txt
+pip install .
```
Comment (makermelissa) [inline install.sh]: What's the reasoning behind this change as well?

**Review by makermelissa — COMMENTED** [review]
I just have a couple of questions regarding some changes that I don't understand the reasoning behind. Thanks.

File: .pre-commit-config.yaml (hunk 1 above)
Comment (tekktrik) [reply .pre-commit-config.yaml:7]: I got an error when trying to use "latest" locally, so I instead pinned it to the latest version the libraries use, but I can double check to see if I misunderstood.

File: install.sh (hunk 2 above)
Comment (tekktrik) [reply install.sh]: Sorry, I don't think this should have changed, I think I was debugging something on another PR - the only thing I can think of is that in the pipeline for `pyproject.toml` there will be `requirements.txt` and a new `optional_requirements.txt` that some libraries will have for optional libraries (like how some of the libraries can use, but don't require, Pillow).  I think there may need to be a solution to that latter, but definitely changing `pip install -r requirements.txt` is not intended.  Sorry for letting it sneak in!

File: .pre-commit-config.yaml (hunk 1 above)
Comment (makermelissa) [reply .pre-commit-config.yaml:7]: Sounds good. What are the other repos doing for this?

Comment (tekktrik) [reply .pre-commit-config.yaml:7]: Other repos are pinning to this version (v0.14.0) of the `reuse` hook.

**Review by makermelissa — APPROVED** [review]
Ok cool. Thanks for double checking. Looks good.

**Pull request merged**